pyproject.toml

```diff
@@ -1,18 +1,25 @@
-[tool.poetry]
+[project]
 name = "habluetooth"
 version = "3.21.0"
+license.text = "Apache-2.0"
```

Comment:
We can't use just `license = "Apache-2.0"` yet since it isn't supported by setuptools. Same for the `dynamic` key, although that it seems to work fine even without it.